source/uwp/Renderer/lib/AdaptiveTextBlock.cpp

```diff
@@ -33,7 +34,8 @@ namespace AdaptiveCards { namespace Rendering { namespace Uwp
     _Use_decl_annotations_
     HRESULT AdaptiveTextBlock::get_Text(HSTRING* text)
     {
-        return UTF8ToHString(m_sharedTextBlock->GetText(), text);
+        DateTimeParser parser(m_sharedTextBlock->GetLanguage());
+        return UTF8ToHString(parser.GenerateString(m_sharedTextBlock->GetTextForDateParsing()), text);
```

Comment:
>return UTF8ToHString(parser.GenerateString(m_sharedTextBlock->GetTextForDateParsing()), text); [](start = 8, length = 94)

Are we sure we want to do this here and not as part of rendering? I'm worried about whether we want someone who calls get_Text on the API to get the raw text or the already processed date. I can't actually think of a scenario where you want to read out the raw text, but it makes me a bit nervous that it's not possible to do so. Maybe double check with Matt if you haven't already just in case.